services/director-v2/src/simcore_service_director_v2/models/domains/projects.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+from typing import Dict, Optional
+
+from models_library.projects import Project, Workbench
+from pydantic import Field
+from pydantic.class_validators import validator
+from pydantic.networks import HttpUrl
+from simcore_postgres_database.models.projects import ProjectType
+
+
+class ProjectAtDB(Project):
```

Comment:
I wonder if we should move these to some models_library package as well ...

Reply:
yeah I thought that if we need to we would. the other problem is that models-library currently does not require postgres-database-library